### ISSUE TITLE
i am not able to get  confidence value through log.i 

### ISSUE THREAD

**Issue by ojhagopal98** [pillarpond/face-recognizer-android#39]
in recognizer.java how can i use or see the confidence value, if I'm not wrong the confidence value is the float value which is displayed under frame in the output screen.
I just want to add that if confidence is greater than 0.6 than email notification should trigger.please help

**Comment by deepsingh132** [comment]
Yes, same question. I'm also not able to fetch the confidence and other values such as string title for person name. Did you find any solution ? I want to display a cardview as soon as the person is recognized, with the person's info and image on the card. but for this i need confidence and other values in my main activity. Any help would be much appreciated !

**Comment by ojhagopal98** [comment]
I used log to display the confidence under multiboxTracker.java file -- >below labelString variable I made a change as Log.i("Tagname", String.valueOf(recognition.detectionConfidence)); through this line, I can see the confidence after running in logcat via Android studio here you can use the confidence value for your application I hope it helps also go through recognizer.java it would help. ATB for your project

**Comment by deepsingh132** [comment]
@ojhagopal98 Thank you for your wishes, but i have some questions, like i want to save the recognized faces in the phone itself and whenever a face is recognized, i want to display it on a cardview as a type of list. Like after 1st person is recognized, display their name and photo on the cardview and add them to the list then same with second and so on. Can you help with that ?
Currently I'm not able to save the recognized images and using preview image for the same.
